src/utils/createChainedFunction.js

```diff
@@ -4,22 +4,26 @@
  * Will only create a new function if needed,
  * otherwise will pass back existing functions or null.
  *
- * @param {function} one
- * @param {function} two
+ * @param {function} functions to chain
  * @returns {function|null}
  */
-function createChainedFunction(one, two) {
-  let hasOne = typeof one === 'function';
-  let hasTwo = typeof two === 'function';
+function createChainedFunction(...funcs) {
+  return funcs
+    .filter(f => f != null)
+    .reduce((acc, f) => {
+      if ({}.toString.call(f) !== '[object Function]') {
```

Comment:
bit of a nitpick, I am a bit confused about the `toString` call? what is it protecting against that `typeof` doesn't catch? it seems very unlikely that anyone is using `new Function` to create a callback. Only mentioning because it strikes me as harder to understand, and React itself doesn't make this check, so I am not sure there is any big need to support such an edge case.